ansible/roles/discourse/tasks/main.yml

```diff
@@ -14,5 +14,25 @@
   template:
     src: standalone.yml.j2
     dest: "{{ dis_path }}/containers/app.yml"
-  notify:
-    launcher rebuild
+  register: dis_template
+
+- name: Get Discourse container info
+  docker_container_info:
+    name: app
+  register: dis_ct
```

Comment:
Can we make the variable named `discourse_container` instead of `dis_ct`? I know it's very long, but at least this way we don't forget in the future what it means